test/utils.py

```diff
@@ -0,0 +1,64 @@
+"""Test utils"""
+import os
+import subprocess
+import sys
+from pathlib import Path
+from typing import Union
+
+import pandas as pd
+
+try:
+    from test.config import ROBOT_JAR_PATH, TEST_IN_DIR, TEST_OUT_DIR, TEST_SPARQL_QEURY_DIR
+except ModuleNotFoundError:
+    from config import ROBOT_JAR_PATH, TEST_IN_DIR, TEST_OUT_DIR, TEST_SPARQL_QEURY_DIR
+
+robot_cmd_options = ['robot', ROBOT_JAR_PATH]
```

Comment:
## Robot usage
I put instructions in the docs for installing `robot` globally. That will be our default. For the GitHub action, I might do this global installation but if it's easier I'll have it download the `.jar`.